Tasks/plane.js

```diff
@@ -1,18 +1,12 @@
 // Make nested array plane
+'use strict';
 
-const_plane = function(arr, res=[]) {
-  j = 0; for(let i = 0, length = arr.length; i < length; i++){
-    value = arr[i];
-    j = i
-    if (Array.isArray(value, typeof value) && [i, length]) {
-    res.push(...const_plane(value));
-    arr[i] = res[i - 1]
-    }else{
-    arr[i] = res[j - 1]
-    res.push(value);
-    };
-  };
-  return res
-}
+const plane = (arr) => {
```

Comment:
```suggestion
const plane = arr => {
```
I think, it looks better without brackets (  )

Reply:
Yes it is acceptable but in this case we need to follow Common hints that say "Use round brackets even for single argument lambda functions"